helix-event/Cargo.toml

```diff
@@ -16,8 +16,8 @@ ahash = "0.8.11"
 hashbrown = "0.14.0"
 tokio = { version = "1", features = ["rt", "rt-multi-thread", "time", "sync", "parking_lot", "macros"] }
 # the event registry is essentially read only but must be an rwlock so we can
-# setup new events on initialization, hardware-lock-elision hugely benefits this case
-# as it essentially makes the lock entirely free as long as there is no writes 
+# setup new events on intalization, hardware-lock-elision hugnly benefits this case
+# as is essentially makes the lock entirely free as long as there is no writes 
```

Comment:
typos

Reply:
also the typo, ‘setup’ (should be ‘set up’)